### PR TITLE
run vtex setup, use ?? on SKUSelector, lint

### PR DESCRIPTION
**Pull request by jgfidelis**
#### What problem is this solving?

- Run vtex setup
- Lint works, run on SKUSelector files
- Use optional chaining on SKUSelector, improve type usage
- Code improvements

#### How should this be manually tested?

https://fidelis--alssports.myvtex.com/patagn-vest-nano-puff/p

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [ ] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: react/components/SKUSelector/types/index.ts [hunk 1]
```diff
@@ -1,20 +1,3 @@
-export interface ProductItem {
```
Comment (jgfidelis) [inline react/components/SKUSelector/types/index.ts]: move to another file

File: react/components/SKUSelector/index.tsx [hunk 2]
```diff
@@ -180,12 +181,12 @@ const SKUSelectorContainer: FC<Props> = ({
   }
 
   const getNewSelectedVariations = useCallback(() => {
-    const hasSKuInQuery = Boolean(query && query.skuId)
```
Comment (jgfidelis) [inline react/components/SKUSelector/index.tsx]: fixing typo

File: react/components/Greeting/index.js [hunk 3]
```diff
@@ -27,7 +27,10 @@ const withWrapper = Component => props => (
 )
 
 const Greeting = ({ orderForm }) => {
-  const firstName = path(['clientProfileData', 'firstName'], orderForm)
+  console.log('teste GREETING')
```
Comment (Klynger) [inline react/components/Greeting/index.js]:
```suggestion
```

File: react/components/Greeting/index.js [hunk 4]
```diff
@@ -27,7 +27,10 @@ const withWrapper = Component => props => (
...
+  const firstName = path(['clientProfileData', 'firstName'], orderForm) || null
+  console.log('teste firstName: ', firstName)
```
Comment (Klynger) [inline react/components/Greeting/index.js]:
```suggestion
```

File: react/components/Greeting/index.js [hunk 5]
```diff
@@ -27,7 +27,10 @@ const withWrapper = Component => props => (
...
+  console.log('teste orderForm: ', orderForm)
```
Comment (Klynger) [inline react/components/Greeting/index.js]:
```suggestion
```

File: react/components/SKUSelector/Wrapper.tsx [hunk 6]
```diff
@@ -21,12 +21,12 @@ const useVariations = (skuItems: ProductItem[], shouldNotShow: boolean, visibleV
...
       for (const currentVariation of skuItem.variations) {
         const { name, values } = currentVariation
-          if (!visibleVariations || visibleVariations.includes(name.toLowerCase().trim())) {
+        if (!visibleVariations || visibleVariations.includes(name.toLowerCase().trim())) {
 
-            const value = values[0]
-            const currentSet = variationsSet[name] || new Set()
-            currentSet.add(value)
-            variationsSet[name] = currentSet
+          const value = values[0]
```
Comment (victorhmp) [inline react/components/SKUSelector/Wrapper.tsx]: Could `values` ever be `undefined`?

Comment (jgfidelis) [reply react/components/SKUSelector/Wrapper.tsx]: if TS says it doesn't... it doesnt :D

**Pull request merged**